src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -404,20 +425,29 @@ def __new_installation_summary_json(self, installation_packages_json):
         self.__refresh_installation_reboot_status()
 
         # Compose substatus message
+        maintenance_run_id = self.execution_config.maintenance_run_id if self.execution_config.maintenance_run_id is not None else ''
+        errors = self.__set_errors_json(self.__installation_total_error_count, self.__installation_errors)
+        substatus_message = self.__compose_installation_substatus_msg(self.execution_config.activity_id, self.__installation_reboot_status, self.__maintenance_window_exceeded, not_selected_patch_count,
+            excluded_patch_count, pending_patch_count, installed_patch_count, failed_patch_count, installation_packages_json, self.execution_config.start_time,
+            self.env_layer.datetime.timestamp(), maintenance_run_id, errors)
+
+        return substatus_message
+
+    def __compose_installation_substatus_msg(self, activity_id, reboot_status, maintenance_window, not_selected, excluded, pending, installed, failed, installation_packages, start_time, last_modified_time, maintenance_id, errors):
         return {
-            "installationActivityId": str(self.execution_config.activity_id),
-            "rebootStatus": str(self.__installation_reboot_status),
-            "maintenanceWindowExceeded": self.__maintenance_window_exceeded,
-            "notSelectedPatchCount": not_selected_patch_count,
-            "excludedPatchCount": excluded_patch_count,
-            "pendingPatchCount": pending_patch_count,
-            "installedPatchCount": installed_patch_count,
-            "failedPatchCount": failed_patch_count,
-            "patches": installation_packages_json,
-            "startTime": str(self.execution_config.start_time),
-            "lastModifiedTime": str(self.env_layer.datetime.timestamp()),
-            "maintenanceRunId": str(self.execution_config.maintenance_run_id) if self.execution_config.maintenance_run_id is not None else '',
-            "errors": self.__set_errors_json(self.__installation_total_error_count, self.__installation_errors)
+            "installationActivityId": str(activity_id),
+            "rebootStatus": str(reboot_status),
+            "maintenanceWindowExceeded": maintenance_window,
+            "notSelectedPatchCount": not_selected,
+            "excludedPatchCount": excluded,
+            "pendingPatchCount": pending,
+            "installedPatchCount": installed,
+            "failedPatchCount": failed,
+            "patches": installation_packages,
+            "startTime": str(start_time),
+            "lastModifiedTime": str(last_modified_time),
```

Comment:
Check and confirm what happens if last_modified_time passed in to this function is already a str

Reply:
both status files yields the same result:
![image](https://github.com/Azure/LinuxPatchExtension/assets/127874208/209d793b-3ea6-4007-84d9-9947b802d846)
![image](https://github.com/Azure/LinuxPatchExtension/assets/127874208/1e02f7da-aa10-4b33-9e41-15f95295f3e5)